draft-ietf-mops-streaming-opcons.md

```diff
@@ -34,6 +34,12 @@ author:
     org: Tencent America LLC
     country: United States of America
     email: spencerdawkins.ietf@gmail.com
+ -
+    ins: M. Stock
```

Comment:
Thanks for submitting this text!  Sorry for the slow review.
We've asked the chairs to weigh in on adding you as an author, hopefully will have some offline conversation on that shortly.